admin/app/com/lucidchart/piezo/admin/controllers/HealthCheckController.scala

```diff
@@ -0,0 +1,27 @@
+package com.lucidchart.piezo.admin.controllers
+
+import org.joda.time.{DateTime,Minutes}
+import org.joda.time.format.ISODateTimeFormat
+import play.api._
+import play.api.Play.{current,configuration}
+import play.api.mvc._
+import scala.io.Source
+
+object HealthCheckController extends Controller {
```

Comment:
The other controllers do not have controller in the name because it is already in the namespace.